package.json

```diff
@@ -25,7 +25,7 @@
 		"@guardian/identity-auth-frontend": "1.0.0",
 		"@guardian/libs": "^15.6.4",
 		"@guardian/shimport": "^1.0.2",
-		"@guardian/source-foundations": "^13.0.0",
+		"@guardian/source-foundations": "^13.1.0",
```

Comment:
I wish this could be pinned [as per our recommendations](https://github.com/guardian/recommendations/blob/main/dependencies.md#javascript).